README.md

```diff
@@ -0,0 +1,185 @@
+ROLE ICINGA2_WEB
+================
+
+[![image](https://img.shields.io/github/license/adfinis-sygroup/ansible-role-icinga2_web.svg?style=flat-square)](https://github.com/adfinis-sygroup/ansible-role-icinga2_web/blob/master/LICENSE)
+
+[![image](https://img.shields.io/travis/adfinis-sygroup/ansible-role-icinga2_web.svg?style=flat-square)](https://travis-ci.org/adfinis-sygroup/ansible-role-icinga2_web)
+
+[![image](https://img.shields.io/badge/galaxy-adfinis--sygroup.icinga2_web-660198.svg?style=flat-square)](https://galaxy.ansible.com/adfinis-sygroup/icinga2_web)
```

Comment:
- New release will be in the `adfinis` namespace rather than `adfinis-sygroup`
- Galaxy NG changed its URLs 🤦 
- Using Github actions rather than Travis
```suggestion
[![image](https://img.shields.io/github/license/adfinis/ansible-role-icinga2_web.svg?style=flat-square)](https://github.com/adfinis/ansible-role-icinga2_web/blob/master/LICENSE)

[![image](https://img.shields.io/github/actions/workflow/status/adfinis/ansible-role-icinga2_web/ansible-ci.yml?branch=master&style=flat-square)](https://github.com/adfinis/ansible-role-icinga2_web/actions/workflows/ansible-ci.yml)

[![image](https://img.shields.io/badge/galaxy-adfinis.icinga2_web-660198.svg?style=flat-square)](https://galaxy.ansible.com/ui/standalone/roles/adfinis/icinga2_client/)
```